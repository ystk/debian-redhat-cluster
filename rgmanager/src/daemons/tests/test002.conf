<?xml version="1.0"?>
<!-- normal resource group -->
<!-- If compared with the inline resource group in test1,
     this should not make this resource group restart since no
     attributes have changed. -->
<cluster>
<rm>
	<resources>
    		<service name="test1"/>
	</resources>
	<service ref="test1"/>
</rm>
</cluster>
