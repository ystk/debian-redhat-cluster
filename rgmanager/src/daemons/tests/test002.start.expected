Starting test1...
[start] service:test1
Start of test1 complete
